Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift

```diff
@@ -30,16 +33,22 @@ internal class PinchGestureHandler: GestureHandler {
             scale = pow(2, delegate.scaleForZoom())
             delegate.gestureBegan(for: .pinch)
 
+            previousPinchCenterPoint = pinchCenterPoint
+
             /**
              TODO: Handle a concurrent rotate gesture here.
              Prioritize the correct gesture by comparing the velocity of competing gestures.
              */
-
         } else if pinchGestureRecognizer.state == .changed {
 
             let newScale = scale * pinchGestureRecognizer.scale
-            delegate.pinchScaleChanged(with: log2(newScale), andAnchor: pinchCenterPoint)
+            let newZoom = log2(newScale)
 
+            let offset = CGSize(width: pinchCenterPoint.x - previousPinchCenterPoint.x,
+                                height: pinchCenterPoint.y - previousPinchCenterPoint.y)
+            previousPinchCenterPoint = pinchCenterPoint
+
+            delegate.pinchChanged(with: newZoom, anchor: pinchCenterPoint, offset: offset)
         } else if pinchGestureRecognizer.state == .ended
             || pinchGestureRecognizer.state == .cancelled {
```

Comment:
I would imagine that the drift logic below also needs to be updated

Reply:
I actually believe none of this drift logic is needed at all anymore. Drag API handles drift for us.